### PR TITLE
Feature/ob 24 reusable dropdown input

### PR DESCRIPTION
**Pull request by Alinochka07**
Task is done

### CODE REVIEW

File: src/Features/header/components/CityLanguageSelect/City/CitySelect.tsx [hunk 1]
```diff
@@ -32,6 +34,13 @@ export const CitySelect = ({ cityValue }: CitySelectProps): ReactElement => {
     setShowModal(false);
   };
 
+  const handleChange = (event: React.MouseEvent<HTMLInputElement>) => {
+    event.preventDefault();
+    setCityInput(event.currentTarget.value);
+    console.log(cityInput);
```
Comment (Alex-2kZharkov) [inline src/Features/header/components/CityLanguageSelect/City/CitySelect.tsx]: remove log

File: src/Features/header/components/CityLanguageSelect/City/CitySelect.tsx [hunk 2]
```diff
@@ -32,6 +34,13 @@ export const CitySelect = ({ cityValue }: CitySelectProps): ReactElement => {
     setShowModal(false);
   };
 
+  const handleChange = (event: React.MouseEvent<HTMLInputElement>) => {
```
Comment (Alex-2kZharkov) [inline src/Features/header/components/CityLanguageSelect/City/CitySelect.tsx]: return type for the function

File: src/components/Dropdown/Dropdown.tsx [hunk 3]
```diff
@@ -0,0 +1,80 @@
+import Link from 'next/link';
+import { ReactElement, PropsWithChildren } from 'react';
+
+export interface InputDropdownProps extends PropsWithChildren {
+  onChange?: (event: any) => void;
```
Comment (Alex-2kZharkov) [inline src/components/Dropdown/Dropdown.tsx]: use something else instead of `any`

File: src/components/Dropdown/Dropdown.tsx [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+  placeholder: string;
+  form_width: string;
```
Comment (Alex-2kZharkov) [inline src/components/Dropdown/Dropdown.tsx]: all these props should be in the `camelCase` notation

File: src/components/Dropdown/Dropdown.tsx [hunk 5]
```diff
@@ -0,0 +1,80 @@
+import Link from 'next/link';
+import { ReactElement, PropsWithChildren } from 'react';
+
+export interface InputDropdownProps extends PropsWithChildren {
```
Comment (Alex-2kZharkov) [inline src/components/Dropdown/Dropdown.tsx]: please check the name of the interface and the component name

**Review by Alex-2kZharkov — CHANGES_REQUESTED** [review]

File: src/components/Dropdown/Dropdown.tsx (hunk 5 above)
Comment (Alinochka07) [reply src/components/Dropdown/Dropdown.tsx]: Renamed. All are titled with the same name.

File: src/components/Dropdown/Dropdown.tsx (hunk 4 above)
Comment (Alinochka07) [reply src/components/Dropdown/Dropdown.tsx]: renamed in the camelCase

File: src/components/Dropdown/Dropdown.tsx (hunk 3 above)
Comment (Alinochka07) [reply src/components/Dropdown/Dropdown.tsx]: indicated the event type. ChangeEvent

File: src/Features/header/components/CityLanguageSelect/City/CitySelect.tsx (hunk 2 above)
Comment (Alinochka07) [reply src/Features/header/components/CityLanguageSelect/City/CitySelect.tsx]: fixed

File: src/Features/header/components/CityLanguageSelect/City/CitySelect.tsx (hunk 1 above)
Comment (Alinochka07) [reply src/Features/header/components/CityLanguageSelect/City/CitySelect.tsx]: done

**Review by Alex-2kZharkov — APPROVED** [review]

**Pull request merged**